Makefile

```
@@ -28,7 +34,9 @@ format-style:
 	autopep8 `git ls-files | grep "\.py$$"`
 
 test: clean check-syntax-errors check-style check-types
-	env PYTEST_ADDOPTS="$$PYTEST_ADDOPTS --cov=openfisca_core" pytest
+	PYTEST_ADDOPTS="$$PYTEST_ADDOPTS ${pytest_addopts}" pytest
+	openfisca test ${COUNTRY_TEMPLATE_TESTS} -c ${COUNTRY_TEMPLATE} ${optional_arguments}
+	openfisca test ${EXTENSION_TEMPLATE_TESTS} -c ${COUNTRY_TEMPLATE} -e ${EXTENSION_TEMPLATE} ${optional_arguments}
 
-api:
-	openfisca serve --country-package openfisca_country_template --extensions openfisca_extension_template
+serve:
+	openfisca serve -c ${COUNTRY_TEMPLATE} -e ${EXTENSION_TEMPLATE} ${optional_arguments}
```

Always prefer long options in scripts to ensure readability. Short options are only syntactic sugar for interactive usage.

```suggestion
	openfisca serve --country-package ${COUNTRY_TEMPLATE} --extensions ${EXTENSION_TEMPLATE} ${optional_arguments}
```